src/flags.rs

```diff
@@ -159,7 +163,7 @@ pub fn set_flags(
   opts.optflag("", "recompile", "Force recompilation of TypeScript code");
   opts.optflag("h", "help", "Print this message");
   opts.optflag("D", "log-debug", "Log debug output");
-  opts.optflag("v", "version", "Print the version");
+  opts.optflagmulti("v", "version", "Print the version.");
```

Comment:
I'm also not sure about `-vv` ... It seems very specific.

I like that we can access this info at runtime via `Deno.versions.gnArgs` (maybe rename this to `Deno.versions.buildArgs`?) ... And I think this is information that most people will not care about.

If we had some way of doing `node -pe` in deno, then we wouldn't need to add a new command-line argument, we could just tell people to to do `deno -pe Deno.versions.gnArgs`. So I'd say for now let's not expose this to the command line, but force people to access it via `Deno.versions.gnArgs`.

(I'm not so happy with how `-pe`  was designed... We should probably discuss in a separate issue.)

Reply:
Maybe we can combine `Deno.versions` and `Deno.platform` into `Deno.build` ?

```
Deno.build.denoVersion
Deno.build.v8Version
Deno.build.typescriptVersion
Deno.build.arch
Deno.build.os
Deno.build.gnArgs
```

Reply:
That's possible, but I think that involves some breaking changes, and I'm not sure that's worth it. (Deno.platform seems being used a few time in deno_std https://github.com/denoland/deno_std/search?q=platform&unscoped_q=platform )

Reply:
I'd rather have breaking changes now than in 6 months. [People have been warned](https://deno.land/manual.html#disclaimer).

I think Deno.versions.gnArgs is not the most semantically meaningful statement... So I will add an issue to do this change.

Reply:
Thanks for making an issue. I'll work on this when the members reach a consensus.